spec/models/validations/setup_validations_spec.rb

```diff
@@ -511,6 +511,59 @@
         expect(record.errors["scheme_id"]).to be_empty
       end
     end
+
+    context "with a scheme with no locations active on the start date & no location set" do
+      let(:scheme) { create(:scheme) }
+      let(:location) { create(:location, scheme:) }
+
+      before do
+        location_deactivation_period = build(:location_deactivation_period, deactivation_date: Time.zone.local(2022, 6, 4), location:)
+        location_deactivation_period.save!(validate: false)
+        location.reload
+      end
+
+      it "produces error when scheme does not have any active locations on the tenancy start date" do
+        record.startdate = Time.zone.local(2022, 7, 5)
+        record.scheme = scheme
+        setup_validator.validate_scheme(record)
+        expect(record.errors["startdate"]).to include(match I18n.t("validations.setup.startdate.scheme.locations_inactive.startdate", name: scheme.service_name))
```

Comment:
Would it make sense to also test that error gets set on `scheme_id` ?

Reply:
Yes definitely will add that, now that I've added the error message to scheme_id also.